l10n_it_fatturapa_sale/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* Simone Rubino
```

Comment:
Come ho contribuito?

Reply:
Il tuo nome era sul manifest ... 😬 

https://github.com/OCA/l10n-italy/blob/5851822caf964aaf2284312a103d34fcd28a5d29/l10n_it_fatturapa_sale/__manifest__.py#L1

Reply:
Ah ecco, grazie, però non penso sia corretto aggiungermi in questa PR visto che non ho fatto nulla; mi potresti togliere?

Reply:
Come preferisci ..

Reply:
> Ah ecco, grazie, però non penso sia corretto aggiungermi in questa PR visto che non ho fatto nulla; mi potresti togliere?

già che ci sei, puoi fare una review veloce?

Reply:
Grazie

> già che ci sei, puoi fare una review veloce?

Fatta https://github.com/OCA/l10n-italy/pull/2983#pullrequestreview-1156244386